packages/@react-aria/virtualizer/src/ScrollView.tsx

```diff
@@ -102,6 +102,8 @@ export function useScrollView(props: ScrollViewProps, ref: RefObject<HTMLElement
         state.isScrolling = true;
         setScrolling(true);
 
+        // Pause typekit MutationObserver during scrolling.
+        window.dispatchEvent(new Event('tk.disconnect-observer'));
```

Comment:
Do we want to move this to a Spectrum-only location? We would currently need to copy it into several places in both v3 and s2, and any custom components people build with virtualizer would also need it. Not sure if there's really much harm for it being there outside Adobe too?

Reply:
I think it is fine keeping it here, like you said I don't think firing this event even for non-spectrum virtualizer experiences is a problem. This event is specific to our typekit right?

Reply:
it would affect any typekit even ones used by third parties, which is probably a good thing

Reply:
Agreed, better to have it included for non-spectrum typekits and also non-spectrum virtualizer setups.

We could do a check for a typekit on load, but I don't think it would make much of a difference.